templates/_tag.tt

```diff
@@ -0,0 +1,8 @@
+<section>
+    <h3><a href="[% plerd.tag_uri(tag) %]">[% tag %]</a></h3>
```

Comment:
As written, this causes a tag-specific page to link to itself via its sole header, which is uneccesary.